openssl/src/ssl/mod.rs

```diff
@@ -570,6 +570,14 @@ impl SslContext {
             })
     }
 
+    #[cfg(feature = "ecdh_auto")]
+    pub fn set_ecdh_auto(&mut self, onoff: bool) -> Result<(),SslError> {
```

Comment:
Can you add a quick bit of documentation to this, saying what it does and mentioning that it requires the `ecdh_auto` feature?